packages/web/components/swap-tool/index.tsx

```diff
@@ -290,622 +296,618 @@ export const SwapTool: FunctionComponent<{
 
     return (
```

Comment:
Is there any way this component can be broken into simpler parts? It's kind of long and hard to follow

Reply:
Yes I agree it's large but IMO breaking this up could be more confusing because then it's more difficult to track what the smaller components are used for. For sake of time, I'll leave it here and we can consider a refactor later?

Reply:
SOunds good